astropy/units/tests/test_quantity_ufuncs.py

```diff
@@ -711,20 +711,16 @@ def test_onearg_test_ufuncs(self, ufunc):
         assert out.dtype == bool
         assert np.all(out == ufunc(q.value))
 
+    # Ignore RuntimeWarning raised on Windows and s390.
+    @pytest.mark.filterwarnings('ignore:.*invalid value encountered in sign')
```

Comment:
The answers are actually reasonable - and those are still tested.  All this does is to filter out the numpy warnings, which we have no control over (and do not occur on all platforms), so that these do not cause test failures.